File: src/DashboardUI/src/data-contracts/ReviewerEstimateConsumptiveUseRequest.ts
import { ApplicationStoreRequestBase } from './ApplicationStoreRequestBase';
import { MapPoint } from './MapPoint';
import { MapPolygon } from './MapPolygon';

export interface ReviewerEstimateConsumptiveUseRequest extends ApplicationStoreRequestBase {
  $type: 'ReviewerEstimateConsumptiveUseRequest';
  waterConservationApplicationId: string;
  polygons: MapPolygon[];
  controlLocation: MapPoint;
  updateEstimate: boolean;
}

File: src/API/WesternStatesWater.WestDaat.Contracts.Client/Requests/Conservation/ReviewerEstimateConsumptiveUseRequest.cs
﻿namespace WesternStatesWater.WestDaat.Contracts.Client.Requests.Conservation;

public class ReviewerEstimateConsumptiveUseRequest : ApplicationStoreRequestBase
{
    public Guid WaterConservationApplicationId { get; set; }

    public MapPolygon[] Polygons { get; set; }

    public MapPoint ControlLocation { get; set; }

    public bool UpdateEstimate { get; set; }
}
